README.md

```diff
@@ -35,18 +35,20 @@ to install different dependencies.
 
 ### Installing dependencies from package repositories
 
-| Dependency | Name of Debian package |
-| ---------- | ---------------------- |
-| [Judy](http://judy.sourceforge.net/) | libjudy-dev |
-| [readline](https://tiswww.case.edu/php/chet/readline/rltop.html) | libreadline-dev |
-| valgrind | valgrind |
-| libtool binary | libtool-bin |
-| Boost library | libboost-dev libboost-system-dev libboost-thread-dev |
+| Dependency | Debian | Fedora |
+| ---------- | --- | --- |
+| [Judy](http://judy.sourceforge.net/) | libjudy-dev | Judy-devel |
+| [readline](https://tiswww.case.edu/php/chet/readline/rltop.html) | libreadline-dev | readline-devel |
+| valgrind | valgrind | valgrind |
+| libtool binary | libtool-bin | libtool |
+| Boost library | libboost-dev libboost-system-dev libboost-thread-dev | boost-devel boost-system boost-thread |
+| [gRPC](https://github.com/grpc/grpc) | | grpc-devel grpc-plugins |
+| [Protobuf](https://github.com/google/protobuf) | | protobuf-devel |
```

Comment:
specify `Install from source (see below)` in the Debian column for these
I don't know if the official Protobuf / gRPC version will work for arbitrary Fedora versions

Reply:
> specify Install from source (see below) in the Debian column for these

Thanks, I've updated the pull request.

> I don't know if the official Protobuf / gRPC version will work for arbitrary Fedora versions

I can confirm that it works with the versions currently provided on Fedora 34 - Protobuf 3.14.0 and gRPC 1.26.0.